### ISSUE TITLE
Keyboard shortcuts not working in chrome input fields

### ISSUE THREAD

**Issue by ChucKN0risK** [xiaokaike/vue-color#143]
I can't copy/paste text inside the `.vc-input__input` dom element. 

How could I enable this behavior?

Thanks in advance 😉

**Comment by linx4200** [comment]
works fine in our [demo page](http://xiaokaike.github.io/vue-color/).

**Comment by ChucKN0risK** [comment]
Indeed... It means that's on me. Thanks anyway. You can close the issue ;)

**Comment by ChucKN0risK** [comment]
I managed to fix the issue. I am currently building an electron app and the clipboard's actions (Copy/Paste/Cut) are not enabled by default. To enable that I followed the instructions in this post 😎
➡️ https://pracucci.com/atom-electron-enable-copy-and-paste.html
